src/main/java/Command.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Command takes in the parsed input from the user and splits it into three categories.
+ * They are the command type, description of the task and the date. Depending on the
+ * type of command, description and date may be empty. Command then returns the
+ * appropriate mesage to return to Logic class to output to the user.
+ *
+ * @param "commandInput"  type of command.
+ * @param "descriptionInput" Y description of task.
+ * @param "dateInput" event date/deadline.
+ * @return output message to the user.
+ */
+public class Command extends Logic {
+    protected Boolean isBye;
+    protected String command;
+    protected String description;
+    protected String date;
+    protected String returnString;
+
+    public Command(String commandInput, String descriptionInput, String dateInput) {
+        this.command = commandInput;
+        this.description = descriptionInput;
+        this.date = dateInput;
+        this.isBye = false;
+    }
+
+    /**
+     * Executes the given command received from the Parser class.
+     * Returns the appropriate message to the Logic class.
+     *
+     * @return output message to the user.
+     */
+    public String executeCommand() {
+        switch (command) {
+        case "bye":
+            isBye = true;
+            returnString = Message.printExit();
+            break;
+        case "todo":
+            if (lIST_INDEX < LIST_LENGTH) {
+                Tasks[lIST_INDEX] = new Todo(description);
+                lIST_INDEX++;
+                returnString = "Got it. I've added this task:\n" + Tasks[lIST_INDEX - 1] + "\n"
+                        + "Now you have " + String.valueOf(lIST_INDEX) + " tasks in the list.";
+            } else {
+                //>= 100 tasks
+                returnString = "List is full! :(";
+            }
+            break;
+        case "deadline":
+            if (lIST_INDEX < LIST_LENGTH) {
+                Tasks[lIST_INDEX] = new Deadline(description, date);
+                lIST_INDEX++;
+                returnString = "Got it. I've added this task:\n" + Tasks[lIST_INDEX - 1] + "\n"
+                        + "Now you have " + String.valueOf(lIST_INDEX) + " tasks in the list.";
+            } else {
+                //>= 100 tasks
+                returnString = "List is full! :(";
```

Comment:
Try not to have different levels of abstraction in the cases. (e.g. create a addTask function instead of making a new String every time.